### PR TITLE
Featre/test env

### PR DESCRIPTION
**Pull request by xahgmah**
Create django application to be able render videojs player without OpenEdx

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,6 +54,7 @@ coverage*.json
 # Django stuff:
 *.log
 local_settings.py
+db.sqlite3
```
Comment (z4y4ts) [inline .gitignore]: You've added it to repo. Please remove.

File: videojs_dev_app/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Development environment for VideoJS player
+
+Simply Django Application for improving VideoJS player. Edx doesn't needed. 
+
+## Setup and run
+ 
+ ```shell
+ pip install -r test_requirements
+ cd videjs_dev_app/
```
Comment (z4y4ts) [inline videojs_dev_app/README.md]: videojs_dev_app

File: videojs_dev_app/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ python manage.py runserver 0.0.0.0:8000
```
Comment (z4y4ts) [inline videojs_dev_app/README.md]: These instructions doesn't work for me.
I'm getting:
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/django/utils/autoreload.py", line 229, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 107, in inner_run
    autoreload.raise_last_exception()
  File "/usr/local/lib/python2.7/site-packages/django/utils/autoreload.py", line 252, in raise_last_exception
    six.reraise(*_exception)
  File "/usr/local/lib/python2.7/site-packages/django/utils/autoreload.py", line 229, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named video_xblock
```

File: .gitignore (hunk 1 above)
Comment (xahgmah) [inline .gitignore]: > You've added db.sqlite3 file to the repo. Please remove.

Done

File: videojs_dev_app/wsgi.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+"""
```
Comment (z4y4ts) [inline videojs_dev_app/wsgi.py]: Why do we need wsgi.py? I thought videojs_dev_app is supposed to be run in dev mode only.

File: videojs_dev_app/views.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            '<h1>404 Error. Unsupported player name. Choose one from list: %s</h1' % PLAYER_LIST
+        )
+    player = player_cls(data)
+    response = player.get_player_html(
+        url=data['href'],
+        autoplay=False,
+        account_id='123',
+        player_id='321',
+        video_id=player.media_id(data['href']),
+        video_player_id='video_player_{}'.format('block_id'),  # pylint: disable=no-member
```
Comment (z4y4ts) [inline videojs_dev_app/views.py]: Do you really need pylint disable here? It doesn't complain to me.

File: videojs_dev_app/views.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        save_state_url='',
+        player_state={
+            'currentTime': 0,
+            'transcripts': []
+        },
+        start_time=0,  # pylint: disable=no-member
+        end_time=0,  # pylint: disable=no-member
```
Comment (z4y4ts) [inline videojs_dev_app/views.py]: Same here about pylint

File: videojs_dev_app/wsgi.py (hunk 4 above)
Comment (xahgmah) [inline videojs_dev_app/wsgi.py]: > Why do we need wsgi.py? I thought videojs_dev_app is supposed to be run in dev mode only.

Was generated on `django-admin startproject` command

File: videojs_dev_app/settings.py [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+# Application definition
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
```
Comment (xahgmah) [inline videojs_dev_app/settings.py]: > Do we really need all these applications?

Was generated on `django-admin startproject` command.
Deleted

**Comment by z4y4ts** [comment]
We'll use xblock-sdk workbench instead. See #195 for the reference.

**Pull request closed without merging**